weblate/templates/footer.html

```diff
@@ -3,12 +3,12 @@
 <footer class="flip pull-left">
  <ul>
   <li>{% blocktrans %}Powered by <a href="{{ weblate_url }}">Weblate {{ version }}</a>{% endblocktrans %}</li>
-  <li><a href="{% url 'about' %}">{% blocktrans %}About Weblate{% endblocktrans %}</a></li>
+  <li><a href="{% url 'about' %}">{% blocktrans %}About{% endblocktrans %}</a></li>
   {% if legal %}<li><a href="{% url 'legal:index' %}">{% trans "Legal" %}</a></li>{% endif %}
   {% if status_url %}<li><a href="{{ status_url }}">{% trans "Service status" %}</a></li>{% endif %}
-  <li><a href="{% url 'contact' %}">{% trans "Contact us" %}</a></li>
+  <li><a href="{% url 'contact' %}">{% trans "Contact" %}</a></li>
   <li><a href="{% documentation 'user/index' %}">{% trans "Documentation" %}</a></li>
-  <li><a href="{{ donate_url }}">{% trans "Donate to Weblate!" %}</a></li>
+  <li><a href="{{ donate_url }}">{% trans "Donate" %}</a></li>
```

Comment:
Again these were intentional to differentiate between donation to Weblate and donation to project which is running Weblate. I wanted to avoid confusion when some third party is running Weblate and clarify that this donation is directed to Weblate and not to that entity. 

For example on https://l10n.opensuse.org/, the Donate link could as well be expected to point to openSUSE...